src/CoreBundle/private/modules/MTTableEditor/views/includes/sharedJS.inc.php

```diff
@@ -71,87 +71,12 @@
         ?>
         var sCurrentRecordName = '<?=str_replace("'", "\'", htmlspecialchars_decode($oTable->GetName())); ?>';
 
-        <?php
-        if ($bRevisionManagementActive) {
-            ?>
-        function AddNewRevision() {
-            CreateModalDialogFromContainer('addNewRevisionDialog');
-            $('#cmsrevisionname').val(sCurrentRecordName);
-        }
-
-        function SaveNewRevision() {
-            document.cmseditform.elements['module_fnc[contentmodule]'].value = 'AddNewRevision';
-            $("#cmseditform").prepend('<div id="hiddenTmpFormFields" style="display:none;"></div>');
-            $("#cmsrevisionname").clone().prependTo("#hiddenTmpFormFields");
-            var cmsrevisiondescriptionTmp = document.addNewRevisionForm.elements['cmsrevisiondescription'].value;
-            $("#cmsrevisiondescription").clone().prependTo("#hiddenTmpFormFields");
-            document.getElementById('cmsrevisiondescription').value = cmsrevisiondescriptionTmp;
-            $('#cmseditform').submit();
-        }
-
-
-        function ActivateRecordRevision(id) {
-            $("#dialog-confirm").dialog({
-                resizable:false,
-                height:180,
-                width:300,
-                modal:true,
-                buttons:{
-                    '<?=TGlobal::Translate('chameleon_system_core.record_revision.action_confirm_restore_revision'); ?>':function () {
-                        $("#cmseditform").prepend('<input type="hidden" value="' + id + '" name="sRecordRevisionId">');
-                        document.cmseditform.elements['module_fnc[contentmodule]'].value = 'ActivateRevision';
-                        $('#cmseditform').submit();
-                        $(this).dialog('close');
-                    },
-                    '<?=TGlobal::Translate('chameleon_system_core.action.abort'); ?>':function () {
-                        $(this).dialog('close');
-                    }
-                }
-            });
-        }
-            <?php
-        }
-        ?>
-    </script>
-<?php
-if ($bRevisionManagementActive) {
-            ?>
-    <div id="addNewRevisionDialog" style="display:none;">
-        <div class="tableeditcontainer">
-            <form name="addNewRevisionForm" id="addNewRevisionForm" action="">
-                <h1><?=TGlobal::OutHTML(TGlobal::Translate('chameleon_system_core.record_revision.header_new_revision')); ?></h1>
+        /**
+         * @deprecated since 6.3.0 - revision management is no longer supported
+         */
+        function AddNewRevision() {};
```

Comment:
I think we can remove these methods, as they were only defined if revision management is active. WDYT?

Reply:
You are right. I checked the code base if anything relies on it hardcoded? Nope. Alle Code that had calls to these methods is already removed.